docs/source/user_guide/peptides_proteins.rst

```diff
@@ -328,6 +328,63 @@ objects to an EmpiricalFormula using :py:meth:`~.AASequence.getFormula`, as tags
 considered in this case (there exists no formula for them). However, they have
 an influence on :py:meth:`~.AASequence.getMonoWeight` and :py:meth:`~.AASequence.getAverageWeight`!
 
+Applying Fixed or Variable Modifications to Sequences
+*****************************************************
+
+In this tutorial, we will cover a step-by-step guide on how to use the pyopenms library to generate modified peptides from a given amino acid sequence.
+
+.. code-block:: python
+    :linenos:
+    from pyopenms import *
+    
+    # Create an amino acid sequence using the fromString() method of the AASequence class.
+    # In this example, we will use the amino acid sequence "TESTMTECSTMTESTR"
+    sequence = pyopenms.AASequence.fromString("TESTMTECSTMTESTR")
```

Comment:
sequence = poms.AASequence.fromString("TESTMTECSTMTESTR")

Reply:
yep I think you can change it like that